client/public/index.html

```diff
@@ -28,6 +28,10 @@
       href="https://fonts.googleapis.com/css2?family=Audiowide&display=swap"
       rel="stylesheet"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
```

Comment:
I think this one can be deleted, since you import the font within your GlobalStyles.